lab.R

```diff
@@ -89,7 +94,9 @@ data("nytcovcounty")
 #' 
 
 #' 1. *Write a pipe that starts with `nytcovcounty` as input, filters down to California and April 1-August 31, 2020, and assigns the result to a variable `filtered_df`.  Hint: You can compare dates as though they were strings, e.g., `date <= '1980-05-17'` gives you dates on or before May 17, 1980.* 
-#' 
+filtered_df= nytcovcounty %>% 
+    filter(date <= '2020-8-31' & date >='2020-04-01') %>% 
```

Comment:
The ISO standard uses two digits for the month.  I don't think leaving it out will break things here, but it's a good habit to stick to the standard. 
```suggestion
    filter(date <= '2020-08-31' & date >='2020-04-01') %>% 
```